src/SocrataSelectQuery.php

```diff
@@ -47,115 +48,51 @@ public function setEndpoint(Endpoint $endpoint = NULL) {
   /**
    * Execute the query. In our case, this is actually the cURL request.
    *
-   * @param $type string
+   * @param string $type
    *   cURL request type, if not the default. The only acceptable value is
    *   currently 'metadata', but we might want others in the future?
    *
    * @return array
+   *   An array containing headers and response body
    */
   public function execute($type = NULL) {
     $retval = FALSE;
```

Comment:
Since we're refactoring this anyway, let's rename the $retval and $res variables to be more descriptive.